cmd/git-sync/main.go

```diff
@@ -60,6 +61,8 @@ var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", envString("HOME", ""
 	"the root directory for git-sync operations, under which --dest will be created")
 var flDest = flag.String("dest", envString("GIT_SYNC_DEST", ""),
 	"the name of (a symlink to) a directory in which to check-out files under --root (defaults to the leaf dir of --repo)")
+var flErrorFile = flag.String("error-file", envString("GIT_SYNC_ERROR_FILE", ""),
+	"the path to the error file where to dump the most recent error details")
```

Comment:
When the error file path is not provided, would it throw an error for not being able to create an error file? If so, could we make this optional? i.e. if it's not specified, skip creating the error file

Reply:
Thanks. Updated `exportError` function to check if the file path is provided.